### DISCUSSION TITLE
Themenideen

### DISCUSSION THREAD

**Discussion by NiklasSeitherDHBW** [NiklasSeitherDHBW/NLP-CrowdFlow#1] [Ideas]
- Buchungstexte von Transaktionen klassifizieren: Beispiel aus Unternehmen - muss abgeändert werden

- Netflix Flime anhand von Kurzzusammenfassung in Genre einteilen: https://www.kaggle.com/code/aayushmishra1512/netflix-data-analysis-and-visualization/input

- Aktienkursvorhersage mit ML und Stimmungserkennung aus Social Media: https://repositorium.hs-ruhrwest.de/frontdoor/deliver/index/docId/741/file/Masterthesis_Vogt_Digitalversion.pdf

**Comment by CVJR03** [comment]
- cyberbullying:
        - https://www.kaggle.com/code/titanpointe/cyberbullying-tweets-eda-automl-dl-bert
        - https://www.kaggle.com/datasets/aadyasingh55/sexism-detection-in-english-texts
        - https://www.kaggle.com/code/jannesklaas/17-nlp-and-word-embeddings
        - https://www.kaggle.com/code/nadeemkaggle123/sexism-detection-in-text-using-lstm
        - https://www.kaggle.com/code/francescoliveras/sexism-comments-eda-and-model-en-es

**Comment by Jango9850** [comment]
- Restaurant Rating prediction auf den reviews: https://www.kaggle.com/datasets/joebeachcapital/restaurant-reviews
- Side effect causing drug klassification: https://www.kaggle.com/datasets/jithinanievarghese/drugs-side-effects-and-medical-condition
- Crypto News Sentiment/ Polarisierung: https://www.kaggle.com/datasets/oliviervha/crypto-news/data

- Aktienkursvorhersage mit ML und Stimmungserkennung + Entityrecognition (für das betroffene Symbol) aus Social Media (potentzielle erweiterung mit yfinance daten): https://www.kaggle.com/datasets/vivekrathi055/sentiment-analysis-on-financial-tweets
